Makefile

```diff
@@ -14,9 +19,9 @@ NODE_BIN := $(THIS_DIR)/node_modules/.bin
 
 # applications
 NODE ?= node
-NPM ?= $(NODE) $(shell which npm)
+NPM ?= npm
```

Comment:
Was there a reason why it was invoked `node npm` previously? `npm` seems like a perfectly fine standalone command. Just making sure I've not broken some exotic setup :)